### PR TITLE
docs: updated admin partitions instructions

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR  updates the Admin Partitions page with the following changes.

🔍  [Preview](https://consul-8qk84206j-hashicorp.vercel.app/docs/enterprise/admin-partitions)

- Helm command now includes the helm version
- Add `$` to some code examples
- Updated Consul versions to `1.11.2`
- Added `global.name` to the client's `values.yaml` file. The client needs a unique name.
- Added comment in the client's `values.yaml` to help the reader know what steps correspond to the k8s auth method, and admin partition service IP. 
- Added link to the learn tutorial. The second one will be uncommented in a future PR.

### CODE REVIEW

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 1]
```diff
@@ -20,6 +20,10 @@ Admin partitions exist a level above namespaces in the identity hierarchy. They
 
 -> **Preexisting resource nodes and namespaces**: Admin partitions were introduced in Consul 1.11. Resource nodes were not namespaced prior to 1.11. After upgrading to Consul 1.11 or later, all resource nodes will be namespaced.
 
+Learn resources are available to help you get started with Admin Partitions.
```
Comment (trujillo-adam) [inline website/content/docs/enterprise/admin-partitions.mdx:23]:
```suggestion
Learn resources are available to help you get started with admin partitions.
```

**Review by trujillo-adam — APPROVED** [review]
Looks good - one suggestion

**Review by trujillo-adam — APPROVED** [review]

**Review by david-yu — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/577874.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit d4f965e49d0925e80d895eafdd37c0f4acc4d589 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit d4f965e49d0925e80d895eafdd37c0f4acc4d589 onto `release/1.11.x` succeeded!

**Pull request merged**